bot-lax-adaptor.sha1

```diff
@@ -1 +1 @@
-d0da160fd6f289d45f266e6e311f0861351a9224
+961c57c524b0b72315961602fbc9ba7ff874f048
```

Comment:
it doesn't look like elife-tools has support for the new data (`editorEvaluation`, `elifeAssessment`, `recommendationForAuthors`, `publicReviews` etc) that the api-raml change captures, so the bot-lax revision will have to stay the same essentially except for a bumped api-raml revision.

Reply:
@gnott , I'm sure you're familiar with all these, they're more for my benefit:

recommendation for authors:
https://github.com/elifesciences/XML-mapping/blob/a559aaa70ac6239528e9ecabe0d2e69802337a95/elife-1234567890-v2/elife-1234567890-v2.xml#L2396-L2454

public reviews:
https://github.com/elifesciences/XML-mapping/blob/a559aaa70ac6239528e9ecabe0d2e69802337a95/elife-1234567890-v2/elife-1234567890-v2.xml#L2046-L2395

editor's assessment:
https://github.com/elifesciences/XML-mapping/blob/a559aaa70ac6239528e9ecabe0d2e69802337a95/elife-1234567890-v2/elife-1234567890-v2.xml#L2001-L2045